packages/sync-actions/src/utils/copy-empty-array-props.js

```diff
@@ -15,13 +15,49 @@ export default function copyEmptyArrayProps(oldObj = {}, newObj = {}) {
           ...newObj,
           ...nextObject,
         }
+
+        if (key === 'custom') {
+          return {
+            ...merged,
+          }
+        }
+
+        if (Array.isArray(value) && newObj[key] && newObj[key].length >= 1) {
+          /* eslint-disable no-plusplus */
+          for (let i = 0; i < newObj[key].length; i++) {
+            if (
+              !isNil(newObj[key][i]) &&
+              typeof newObj[key][i] === 'object' &&
+              !isNil(newObj[key][i].id)
+            ) {
+              /* eslint-disable eqeqeq */
```

Comment:
what's this sourcery

Reply:
I also thought it looked funny, turns out it's valid: https://eslint.org/docs/rules/eqeqeq